docs/typescript.md

```diff
@@ -0,0 +1,43 @@
+# Lottie React Native and TypeScript
+
+If you are here, you are interested on using `lottie-react-native` in your typescript configured project (or just want to get in typescript).
+
+## Getting started
+
+This documentation assumes you have configured your project following the instructions provided in the [readme file](../README.md), it also assumes you have 
+
+## Usage
+
+The usage of `LottieView` is not that different to its JavaScript counterpart, but depending on how your typescript project is configured you might need to import `LottieView` as defined below:
+
+- Your `tsconfig.json` file defines `"esModuleInterop": false,`. In this case, you are forced to define your import as 
+
+```tsx
+import LottieView = require("lottie-react-native");
+```
+
+- Your `tsconfig.json` file deines `"esModuleInterop": true,` and `"allowSyntheticDefaultImports": true,` (the default is `true`). In this case, you can import lottie as:
+
+```tsx
+import LottieView from "lottie-react-native";
+```
+
+### Sample code
+
+```tsx
+import * as React from 'react';
```

Comment:
if you have synthetic imports on (which you should) then you don’t have to use this spec-violating form, you can `import React from ‘react’`

Reply:
Ok, point taken